### DISCUSSION TITLE
PN532 supplier

### DISCUSSION THREAD

**Discussion by pilot1981** [rednblkx/HomeKey-ESP32#129] [General]
Hi,
i bought PN532 from Aliexpress but It's seems to don't works fine (don't feel apple iphone NFC):

Which is the best supplier do you used?

**Comment by darkside9009** [comment]
Hi,

if you want you can use my hardware, which is directly working with this project.

https://casmo.info/product-details/?product=17

**Reply by rhamblen** [reply]
Got the board and it works perfectly.  BUT i noticed you have 3 leds... one for power but the other two are not labels or documented with a GPIO number... is this documented anywhere? I wanted to use it to show a pass or fail on the HK recognition.

thanks

**Reply by rednblkx** [reply]
LEDs are assigned to pin 25 and pin 26 on that board

**Reply by rhamblen** [reply]
Thanks

**Comment by aksneo** [comment]
> Hi,
> 
> i bought PN532 from Aliexpress but It's seems to don't works fine (don't feel apple iphone NFC):
> 
> 
> 
> Which is the best supplier do you used?

How did you notice ?

**Comment by FezVrasta** [comment]
> Hi,
> 
> 
> 
> if you want you can use my hardware, which is directly working with this project.
> 
> 
> 
> https://casmo.info/product-details/?product=17
> 
> 

Hi,do you plan to restock it?

**Comment by darkside9009** [comment]
Hi,

Hits already online.




------ Originalnachricht ------
Von "Federico Zivolo" ***@***.***>
An "rednblkx/HomeKey-ESP32" ***@***.***>
Cc "darkside9009" ***@***.***>; "Comment" 
***@***.***>
Datum 24.08.2025 08:59:26
Betreff Re: [rednblkx/HomeKey-ESP32] PN532 supplier (Discussion #129)
